app/src/main/java/com/sdp13epfl2021/projmag/UserTypeChoice.kt

```diff
@@ -0,0 +1,32 @@
+package com.sdp13epfl2021.projmag
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.widget.RadioButton
+import androidx.appcompat.app.AppCompatActivity
+import com.sdp13epfl2021.projmag.activities.ProfilePageActivity
+
+class UserTypeChoice : AppCompatActivity() {
+    lateinit var ProfButton: RadioButton
+    lateinit var StuButton: RadioButton
```

Comment:
If theses buttons are only used in onCreate, we could remove theses var and use val profButton = ... in onCreate.
Variable name should start with lowercase for camelCase. (also, Stu is maybe a too short abreviation).